ansible/playbooks/host-setup.yml

```
@@ -0,0 +1,246 @@
+---
+- name: Docker MCP Host Setup and SSH Key Distribution
+  hosts: all
+  gather_facts: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)